### ISSUE TITLE
Duplicated Source Codes - SPLs

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#38]
**VERSION v5.0 21-Mar-2015**

There are duplicated SPLs.  I know you mentioned that SPLs needed some TLC though.

```{sql}
SELECT CONCEPT_CODE, VOCABULARY_ID, DOMAIN_ID, COUNT(*)
FROM CONCEPT
WHERE INVALID_REASON IS NULL
AND VOCABULARY_ID NOT IN (
	'MedDRA','SMQ','DRG'
)
AND CONCEPT_CODE NOT IN (
	'OMOP generated'
) AND DOMAIN_ID='Drug'
GROUP BY CONCEPT_CODE, VOCABULARY_ID, DOMAIN_ID
HAVING COUNT(*) > 1;
```

```{txt}
CONCEPT_CODE							VOCABULARY_ID	DOMAIN_ID	COUNT
7f11be5c-6112-c93e-4d0f-d3a085ac698a	SPL	Drug	2
dc374fa4-350b-425d-a7db-7099fdc64cf1	SPL	Drug	2
3F0F6064-CC9F-491C-A940-3F087ABBC774	SPL	Drug	2
5b8a6767-2d46-401c-9f85-d427dda9eff7	SPL	Drug	2
793F034E-5BF8-4FEA-8AF3-85F164BBCBCC	SPL	Drug	2
a84a73a7-8e3a-4985-8be0-04b3028f5e49	SPL	Drug	2
090660c1-6e6d-457f-adb5-046ddfcd1465	SPL	Drug	2
A0D01539-8413-4703-94CC-D221918630A1	SPL	Drug	2
B151E4EB-BB9A-41B9-98A4-2E12BB4E2E5F	SPL	Drug	2
012ABAD7-D804-4167-BF5C-BE6FC2EAFE10	SPL	Drug	2
7bca1d24-a317-44c4-8db2-68030eb961ed	SPL	Drug	2
CA8A0862-1956-43AD-844D-77153AA09DFB	SPL	Drug	2
01417f60-dbf3-4041-bf3c-51e6c9216f06	SPL	Drug	2
d8cf50d4-0448-4527-a498-cddd511213f0	SPL	Drug	2
9cdd9d7e-f0b9-4d81-8f20-a16e687b479e	SPL	Drug	2
```

**Comment by ericaVoss** [comment]
Just making a note that this still exists in **v5.0 3-Apr-2015** - it is not high priority.

**Comment by ericaVoss** [comment]
*v5.0 21-JAN-16*

There are now 393 CONCEPT_CODES that duplicate (in SPL and GPI).

Here are the worst ones:
```{text}
CONCEPT_CODE	VOCABULARY_ID	DOMAIN_ID	COUNT
70050030002600	GPI	Drug	8
65100055102003	GPI	Drug	10
20100002002000	GPI	Drug	11
17100020201800	GPI	Drug	11
78101000000100	GPI	Drug	19
```

So if I was mapping to that last GPI I would end up with 19 matches.

**Comment by cgreich** [comment]
No such thing in SPL: select concept_code, count(8) from concept where vocabulary_id='SPL' group by concept_code having count(8)>1;
Investigating GPI now.

**Comment by ericaVoss** [comment]
Your query returns 31 rows for me.

Here are some examples:
```{text}
d8cf50d4-0448-4527-a498-cddd511213f0
090660c1-6e6d-457f-adb5-046ddfcd1465
F103B9F9-E081-419D-A2D8-B86ACDA834DD
7F11BE5C-6112-C93E-4D0F-D3A085AC698A
```

**Comment by cgreich** [comment]
Refresh the vocab.

**Comment by ericaVoss** [comment]
We downloaded the copy Friday (v5.0 21-JAN-16) . . . did you release another copy since then?

**Comment by cgreich** [comment]
A!!!!! SQL Server and it's wonderful ignorance of upper and lower case.

You are right. There are duplicates with different cases. We'll fix.

**Comment by ericaVoss** [comment]
Fixed in **v5.0 11-MAR-16**.
